common/test/acceptance/tests/lms/test_lms_cohorted_courseware_search.py

```diff
@@ -0,0 +1,245 @@
+"""
+Test courseware search
+"""
+import os
+import json
+
+from ...pages.common.logout import LogoutPage
+from ...pages.studio.overview import CourseOutlinePage
+from ...pages.lms.courseware_search import CoursewareSearchPage
+from ...fixtures.course import XBlockFixtureDesc
+
+from nose.plugins.attrib import attr
+
+from ..studio.base_studio_test import ContainerBase
+
+from ...pages.studio.settings_group_configurations import GroupConfigurationsPage
+from ...pages.studio.auto_auth import AutoAuthPage as StudioAutoAuthPage
+from ...fixtures import LMS_BASE_URL
+from ...pages.studio.component_editor import ComponentVisibilityEditorView
+from ...pages.lms.instructor_dashboard import InstructorDashboardPage
+
+from bok_choy.promise import EmptyPromise
+
+
+@attr('shard_1')
+class CoursewareSearchCohortTest(ContainerBase):
```

Comment:
Since these work now, I would like to keep them (no harm in some extra testing). I'll add some more unit tests.